pyproject.toml

```diff
@@ -9,3 +9,9 @@ build-backend = 'setuptools.build_meta'
 
 [tool.cibuildwheel]
 skip = "pp*"
+
+[tool.pytest.ini_options]
+filterwarnings = [
+    "ignore:numpy.ndarray size changed:RuntimeWarning",
```

Comment:
Oh hmm looks like astropy no longer ignores it but that might require you to mess with your build settings here, maybe. 🤷‍♀️  https://github.com/astropy/astropy/pull/17057

Otherwise you might need `\\.`

Reply:
Fixed.